sqlmesh/core/snapshot/definition.py

```diff
@@ -510,7 +510,7 @@ def from_node(
         node_audits = (
             tuple(t.cast(_Model, node).referenced_audits(audits or {}))
             if node.is_model
-            else tuple([t.cast(StandaloneAudit, node).audit])
+            else tuple()
```

Comment:
Shouldn't this be `tuple(node)`?

Reply:
This is for attached audits. I can maybe change the name.

Reply:
Refactored to a kwarg, so now only used if the node is a model.